perma_web/api/authorizations.py

```diff
@@ -17,7 +17,9 @@ def can_access(self, user, obj):
         # shared folders
         elif obj.vesting_org_id:
             if user.is_vesting_org_member():
-                return user.vesting_org_id == obj.vesting_org_id
+                for vo in user.vesting_org.all():
```

Comment:
It's probably more efficient to replace these three lines with `return user.vesting_orgs.filter(pk=obj.vesting_org_id).exists()` -- it avoids fetching the data and instantiating an object for each vesting org, just a SQL count query.